libvast/src/msgpack.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright Tenzir GmbH. All rights reserved.
```

Comment:
The header doesn't seem right.

Reply:
(Please check all files again.)

Reply:
Ah right, I forgot about the banner every time.